Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -11,37 +11,58 @@ function cleanup() {
 	pkill gzclient
 }
 
+function spawn_model() {
+	MODEL=$1
+	N=$2 #Instance Number
+
+	if [ "$MODEL" != "iris" ] && [ "$MODEL" != "plane" ] && [ "$MODEL" != "standard_vtol" ]
+	then
+		echo "Currently only the following vehicle models are supported! [iris, plane, standard_vtol]"
+		exit 1
+	fi
+
+	working_dir="$build_path/instance_$n"
+	[ ! -d "$working_dir" ] && mkdir -p "$working_dir"
+
+	pushd "$working_dir" &>/dev/null
+	echo "starting instance $n in $(pwd)"
+	../bin/px4 -i $n -d "$src_path/ROMFS/px4fmu_common" -w sitl_${MODEL}_${n} -s etc/init.d-posix/rcS >out.log 2>err.log &
+	python3 ${src_path}/Tools/sitl_gazebo/scripts/xacro.py ${src_path}/Tools/sitl_gazebo/models/rotors_description/urdf/${MODEL}_base.xacro \
+		rotors_description_dir:=${src_path}/Tools/sitl_gazebo/models/rotors_description mavlink_udp_port:=$(($mavlink_udp_port+$N)) \
+		mavlink_tcp_port:=$(($mavlink_tcp_port+$N))  -o /tmp/${MODEL}_${N}.urdf
+
+	gz sdf -p  /tmp/${MODEL}_${N}.urdf > /tmp/${MODEL}_${n}.sdf
+	echo "Spawning ${MODEL}_${N}"
+
+	gz model --spawn-file=/tmp/${MODEL}_${N}.sdf --model-name=${MODEL}_${N} -x 0.0 -y $((3*${N})) -z 0.0
+
+	popd &>/dev/null
+
+}
+
 if [ "$1" == "-h" ] || [ "$1" == "--help" ]
 then
-	echo "Usage: $0 [-n <num_vehicles>] [-m <vehicle_model>]"
+	echo "Usage: $0 [-n <num_vehicles>] [-m <vehicle_model>] [-w <world>] [-s <script>]"
+	echo "-s flag is used to script spawning vehicles e.g. $0 -s iris:3, plane:2"
```

Comment:
```suggestion
	echo "-s flag is used to script spawning vehicles e.g. $0 -s iris:3,plane:2"
```

Reply:
Thanks! I do handle spaces in the script, so it is still legal to have spaces, but I think it is better to remove them from the example

Reply:
Yes I saw that, but the example was still wrong, because it was passed as separate arguments (missing quotes and exactly the reason why I wanted to avoid spaces :).